examples/llms/providers/groq.ts

```diff
@@ -0,0 +1,21 @@
+import "dotenv/config";
+import { BaseMessage } from "bee-agent-framework/llms/primitives/message";
+import { GroqChatLLM } from "bee-agent-framework/adapters/groq/chat";
+
+const llm = new GroqChatLLM({
+  modelId: "gemma2-9b-it",
+  parameters: {
+    temperature: 0.7,
+    max_tokens: 1024,
+    top_p: 1,
+  },
+});
+
+console.info("Meta", await llm.meta());
+const response = await llm.stream([
+  BaseMessage.of({
+    role: "user",
+    text: "Hello world!",
+  }),
+]);
```

Comment:
Use the `generate` method instead to be consistent with other examples

Reply:
this is done.